src/content/docs/es/guides/bursa/008-key-commands.md

```
@@ -0,0 +1,279 @@
+> **Rutas de derivación por tipo de clave:**
+> -  CIP-1852: root, account, payment, stake (m/1852'/1815'/...)
+> -  CIP-1853: pool-cold (m/1853'/1815'/...)
+> -  CIP-1855: policy (m/1855'/1815'/...)
+> -  CIP-0105: drep, committee-cold, committee-hot (m/1852'/1815'/account'/role/...)
+> -  CIP-88/151: calidus (m/1852'/1815'/account'/0/index, autenticación SPO)
+
+***
+
+<a name="root"></a>
+
+#### Clave raíz
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)